File: src/main/java/gargoyle/ct/convert/impl/LongConverter.java
package gargoyle.ct.convert.impl;

import gargoyle.ct.convert.Converter;
import org.jetbrains.annotations.NotNull;

public class LongConverter implements Converter<Long> {
    @Override
    public String format(Long data) {
        return String.valueOf(data);
    }

    @NotNull
    @Override
    public Long parse(@NotNull String data) {
        return Long.valueOf(data);
    }
}

File: src/main/kotlin/gargoyle/ct/convert/impl/LongConverter.kt
package gargoyle.ct.convert.impl

import gargoyle.ct.convert.Converter

class LongConverter : Converter<Long> {
    override fun format(data: Long): String {
        return data.toString()
    }

    override fun parse(data: String): Long {
        return data.toLong()
    }
}
